app/scripts/controllers/sign.ts

```diff
@@ -414,7 +414,7 @@ export default class SignController extends BaseControllerV2<
    * @param msgId - The id of the message to cancel.
```

Comment:
For completion sake, should we add the return value to the method doc also so it's clear we return the full background state?